### PR TITLE
Driver for SSD1289 LCD

### PR DESCRIPTION
**Pull request by spearson78**

### CODE REVIEW

**Comment by sago35** [comment]
Thank you for your contribution.
Please add Examples.

**Comment by spearson78** [comment]
Sorry. I didn't see the example folder. I've updated the pull request.

**Comment by spearson78** [comment]
I need to add further updates for the readme and smoke test. I'm currently working on the XPT2046 pull request. Once thats completed I'll make all the same structural changes to this driver.

**Comment by spearson78** [comment]
I added a smoke test and line in the Readme.

**Review by sago35 — COMMENTED** [review]
@spearson78 
I think it's good.
Please rebase and fix the conflict.

**Comment by spearson78** [comment]
I've rebased and resolved the conflicts.

File: ssd1289/pinbus.go [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+		pins[i].Configure(machine.PinConfig{Mode: machine.PinOutput})
+	}
+
+	return pinBus{
+		pins: pins,
+	}
+}
+
+func (b pinBus) Set(data uint16) {
+	b.pins[15].Set((data & (1 << 15)) != 0)
```
Comment (deadprogram) [inline ssd1289/pinbus.go:21]: Consider replacing with:

```go
for(i :=0; i < 16; i++) {
    b.pins[i].Set((data & (1 << i)) != 0)
}
```

**Comment by deadprogram** [comment]
Thanks for working on this @spearson78 now merging.

**Pull request merged**